---
title: "Pairing Metagenomics and Metaproteomics to Characterize Ecological Niches and Metabolic Essentiality of Gut Microbiomes"
date: 2024-05-01
pubtype: "Article"
featured: true
description: "<b>Tong Wang</b>*, Leyuan Li*, Daniel Figeys, Yang-Yu Liu, <i><b>ISME Communications</b>, 2024</i>"
tags: #["niche","metaproteome"]
image: #"/img/organicdevops.webp"
link: "https://academic.oup.com/ismecommun/advance-article/doi/10.1093/ismeco/ycae063/7660938"
fact: "Interesting little tidbit shown below image on summary and detail page"
weight: 400
sitemap:
  priority : 0.8
---

The genome of a microorganism encodes its potential functions that can be implemented through expressed proteins. It remains elusive how a protein’s selective expression depends on its metabolic essentiality to microbial growth or its ability to claim resources as ecological niches. To reveal a protein’s metabolic or ecological role, we developed a computational pipeline, which pairs metagenomics and metaproteomics data to quantify each protein’s gene-level and protein-level functional redundancy simultaneously. We first illustrated the idea behind the pipeline using simulated data of a consumer-resource model. We then validated it using real data from human and mouse gut microbiome samples. In particular, we analyzed ABC-type transporters and ribosomal proteins, confirming that the metabolic and ecological roles predicted by our pipeline agree well with prior knowledge. Finally, we performed in vitro cultures of a human gut microbiome sample and investigated how oversupplying various sugars involved in ecological niches influences the community structure and protein abundance. The presented results demonstrate the performance of our pipeline in identifying proteins’ metabolic and ecological roles, as well as its potential to help us design nutrient interventions to modulate the human microbiome.